spec/controllers/target_controller_spec.rb

```diff
@@ -59,6 +60,30 @@
           expect(@parsed_body[0]['edit_url']).to eq(edit_target_url(target_1))
         end
       end
+
+      context 'when a target has a match' do
+        let!(:target_1) { create(:target) }
+        let!(:target_compatible) { create(:target) }
+        it 'returns compatible targets' do
+          target_1.length = 1000
```

Comment:
do this when defining the target on the let!
`let!(:target1) { create :target, latitude: -34.123 ... etc }`